src/components/Home/QR/ResultCheckin/index.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react';
+import {
+  Text, View, ActivityIndicator,
+} from 'react-native';
+import { connect } from 'react-redux';
+import { checkQRDispatch } from 'datalayer/actions/qr.action';
+import Auth from 'utils/auth';
+import CheckInEnum from 'constants/checkInEnum';
+import styles from './index.styles';
+
+class ResultCheckin extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      checkInEnum: null,
+    };
+  }
+
+  componentWillMount() {
+    const { navigation, checkQRDispatch } = this.props;
+    const eventId = navigation.getParam('eventId');
+    const barcodeResult = navigation.getParam('barcodeResult');
+    const uid = Auth.getUserId();
+    checkQRDispatch(barcodeResult, eventId, uid)
+      .then(res => {
+        if (!res.success) {
+          /**
+           * @todo Specific errors will be declared here.
+           * @todo Depends on how many type of errors that API response.
+           */
```

Comment:
This is OK but `// TODO: something` is enough, with single line comment. New IDE like VSCODE can help tracking TODOs like that.

Reply:
sorry my bad